build.cake

```diff
@@ -8,10 +8,12 @@ var framework = Argument<string>("framework", "netstandard2.0");
 var runtime = Argument<string>("runtime", "win-x64");
 var sln = "./GraphQL.Server.Transports.sln";
 var projectFiles = new [] {
-  "./src/AspNetCore/AspNetCore.csproj",
-  "./src/WebSockets/WebSockets.csproj",
+  "./src/Transports.AspNetCore/Transports.AspNetCore.csproj",
+  "./src/Transports.Subscriptions.Abstractions/Transports.Subscriptions.Abstractions.csproj",
+  "./src/Transports.Subscriptions.WebSockets/Transports.Subscriptions.WebSockets.csproj",
   "./src/Ui.Playground/Ui.Playground.csproj",
-  "./src/Ui.GraphiQL/Ui.GraphiQL.csproj"
+  "./src/Ui.GraphiQL/Ui.GraphiQL.csproj",
+  "./src/Ui.Voyager/Ui.Voyager.csproj"
```

Comment:
I believe we need to add `GraphQL.Server.Core` to this list so it gets built at as Nuget package?

Reply:
It’s not referenced as a NuGet package by the other projects, just as a project reference. So the Core dll should be included the nupkg for the projects that reference it. 

I don’t see much benefit in making it a separate package, but either way should work.

Reply:
I’m fine with it being included as well. Though if you download the packaged nugets from AppVeyor it is not included.  So need to fix that somehow.

Reply:
I think it should be a NuGet package. Otherwise people will run into problems with different versions of it when installing more than one package including it.

Reply:
Yeah, that's true. Do you need me to do anything on this?

Reply:
Adding Core to packaging